harvest/harvest/pipelines.py

```diff
@@ -27,7 +28,7 @@ def __indico(self, indico_id):
 
         sponsor = None
         try:
-            sponsor = data["results"][0]["organizer"].split("/\\-")[0].strip()
+            sponsor = re.split(r"[\/|-]", data["results"][0]["organizer"])[0].strip()
```

Comment:
`r` corresponds to `raw string`. Otherwise, it gives me `W605 invalid escape sequence '\/'`